ghost/content.md

```diff
@@ -1,33 +1,58 @@
 # Ghost
 
-Ghost is a free and open source blogging platform written in JavaScript and distributed under the MIT License, designed to simplify the process of online publishing for individual bloggers as well as online publications.
+Ghost is a free and open source blogging platform written in JavaScript and distributed under the MIT Licence, designed to simplify the process of online publishing for individual bloggers as well as online publications.
 
 > [wikipedia.org/wiki/Ghost_(blogging_platform)](http://en.wikipedia.org/wiki/Ghost_%28blogging_platform%29)
 
 %%LOGO%%
 
 # How to use this image
 
+This will start a Ghost instance listening on the default Ghost port of 2368.
+
 ```console
-$ docker run --name some-ghost -d ghost
+$ docker run —d --name ghostdemo ghost
```

Comment:
While that is true to an extent @SISheogorath is correct, it wouldn't be published - if you went directly to the IP of the container on the Docker network interface it would be available on port 2368, however this is not how you usually access containers as that IP address will be completely unreliable, and will only be accessible locally by default.